### PR TITLE
feat: Upgrade to Keptn 0.17

### PR DESCRIPTION
**Pull request by TannerGabriel**
## This PR

- Change Keptn install Helm values
- Update distributor to `0.17.0`
- Update `go-utils` to `0.17.0`
- Add new release in README

Integration Tests: https://github.com/keptn-contrib/prometheus-service/actions/runs/2689800969

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,32 +152,42 @@ You can customize prometheus-service with the following environment variables:
...
+    - **
+      throughput**: `sum(rate(http_requests_total{job="<service>-<project>-<stage>-canary"}[<test_duration_in_seconds>s]))`
+    - **
+      error_rate**: `sum(rate(http_requests_total{job="<service>-<project>-<stage>-canary",status!~'2..'}[<test_duration_in_seconds>s]))/sum(rate(http_requests_total{job="<service>-<project>-<stage>-canary"}[<test_duration_in_seconds>s]))`
+    - **
+      response_time_p50**: `histogram_quantile(0.50, sum(rate(http_response_time_milliseconds_bucket{job='<service>-<project>-<stage>-canary'}[<test_duration_in_seconds>s])) by (le))`
+    - **
+      response_time_p90**: `histogram_quantile(0.90, sum(rate(http_response_time_milliseconds_bucket{job='<service>-<project>-<stage>-canary'}[<test_duration_in_seconds>s])) by (le))`
+    - **
+      response_time_p95**: `histogram_quantile(0.95, sum(rate(http_response_time_milliseconds_bucket{job='<service>-<project>-<stage>-canary'}[<test_duration_in_seconds>s])) by (le))`
```
Comment (christian-kreuzberger-dtx) [inline README.md:183]: Pretty sure those changes were not intended, but done by the IDE :D however, they kill the formatting:
image

**Review by christian-kreuzberger-dtx — CHANGES_REQUESTED** [review]
just left a comment on the readme, rest seems fine

File: README.md (hunk 1 above)
Comment (TannerGabriel) [reply README.md]: Sorry, I forgot to render the file after the formatting. I just reversed the changes.

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**